homeassistant/components/media_player/__init__.py

```diff
@@ -705,6 +730,11 @@ def support_shuffle_set(self):
         """Boolean if shuffle is supported."""
         return bool(self.supported_features & SUPPORT_SHUFFLE_SET)
 
+    @property
+    def support_repeat_set(self):
```

Comment:
This is not used.

Reply:
The media_player component has helpers for testing most support bits. They don't seem to be used much though (if at all) so I will remove my addition.